dbt/adapters/athena/connections.py

```diff
@@ -74,6 +75,10 @@ def type(self) -> str:
     def unique_field(self) -> str:
         return f"athena-{hashlib.md5(self.s3_staging_dir.encode()).hexdigest()}"
 
+    @property
+    def get_effective_num_retries(self) -> int:
```

Comment:
```suggestion
    def effective_num_retries(self) -> int:
```
as this is a property, let's use corresponding naming